x-pack/plugins/observability/public/components/shared/types.ts

```diff
@@ -7,7 +7,7 @@
 
 import { ReactNode } from 'react';
 import { AppMountParameters } from '@kbn/core/public';
-import { UXMetrics } from './core_web_vitals';
+import { UXMetrics } from '@kbn/observability-shared-plugin/public';
```

Comment:
Why should the `UXMetrics` type live in Obs Shared? It's a type maintained by the Synthetics team?

Reply:
UXMatrics relates to core web vitals component in observability overview page and UX plugin.

Reply:
It doesn't make sense to me that a type which is produced by one app should go into Observability Shared.

It can just as easily be exported by the UX Plugin and then imported into the Obs app in the place where it is needed.

Reply:
i mentioned due to circular dependency issues. this can't be done at the moment.